frontend/src/components/FormEndPage/EndPageBlock.tsx

```diff
@@ -96,18 +107,25 @@ export const EndPageBlock = ({
             {submissionTimestamp}
           </Text>
         </Box>
-        <Box mt="2.25rem">
-          {isButtonHidden || (
-            <Button
-              as="a"
-              href={endPage.buttonLink || window.location.href}
-              variant="solid"
-              colorScheme={`theme-${colorTheme}`}
-            >
-              {endPage.buttonText || 'Submit another response'}
-            </Button>
-          )}
-        </Box>
+
+        {form?.responseMode == FormResponseMode.Multirespondent &&
+        form?.hasStatusTracker ? (
+          <Box mt="2rem">
+            <StatusTrackerLink
+              formId={form?._id}
+              submissionId={submissionData.id}
+            />
+          </Box>
+        ) : (
+          <Box mt="2.25rem">
+            {isButtonHidden || (
+              <SubmitAnotherResponseButton
+                endPage={endPage}
+                colorTheme={colorTheme}
+              />
+            )}
+          </Box>
+        )}
```

Comment:
just wanna make sure I understand this correctly:
- when a form has status tracking enabled
  - i.e. the **responseMode** is **Multirespondent** and
  - **hasStatusTracker** is `true`
- then the net effect is the "Submit Another Response" button moves to a different place in the DOM
  - from the end of [FormEndPage > EndPageBlock](https://github.com/opengovsg/FormSG/pull/8469/files/65940088d18572fc27c3c2f0f79fd4f9652b4e55#diff-1aa34068d00565c326c8faf1d4bb574296adc3002ef6dbfd5afe7a4b2af9a2d9R111-R128)
  - to at the [end of FormEndPage](https://github.com/opengovsg/FormSG/pull/8469/files/65940088d18572fc27c3c2f0f79fd4f9652b4e55#diff-d7c2fce9e7d368eee6920b5843b0e2cc5b8f01032bdbd26f71f114b5519262e8R59-R65)